content/en/containers/guide/ad_identifiers.md

```diff
@@ -13,13 +13,15 @@ further_reading:
   text: "Manage which Container to include in the Agent Autodiscovery"
 ---
 
-Autodiscovery container identifiers, or `ad_identifiers`, allow you to apply an Autodiscovery configuration file template to a given container, either by using the [container short image](#short-image-container-identifiers) or with a [custom Autodiscovery container identifier](#custom-autodiscovery-container-identifiers).
+Autodiscovery container identifiers, or `ad_identifiers`, allow you to apply an Autodiscovery configuration file template to a given container, either by using the container image name, or by using a custom Autodiscovery container identifier.
 
-**Note**: For other configuration types—key-value stores, Docker labels, or Kubernetes pod annotations—the matching between an integration configuration template and the corresponding container is based on the `<CONTAINER_IDENTIFIER>` included in the key-value stores, labels, or annotations configuration.
+Even if Autodiscovery configuration is defined within a custom configuration file, you can use the standard labels for tagging `env`, `service`, and `version`. See [Unified Service Tagging][1] for more information on how to configure these labels on your containers.
 
-## Short image container identifiers
+**Note**: For other configuration types—key-value stores, Docker labels, or Kubernetes pod annotations—the matching between an integration configuration template and the corresponding container is based on the `<CONTAINER_IDENTIFIER>` included in the key-value stores, labels, or annotations.
```

Comment:
The fact that a dash used for emphasis and a dash in a compound word are next to each other in "types-key-value" makes this sentence a bit confusing. Maybe a reword would help?

```suggestion
**Note**: Other configuration types, including key-value stores, Docker labels, or Kubernetes pod annotations, use a different method to match integration configuration templates to their corresponding containers. For those configuration types, the matching between an integration configuration template and the container is based on the `<CONTAINER_IDENTIFIER>` included in the key-value stores, labels, or annotations.
```